openedx/features/specializations/views.py

```diff
@@ -38,8 +40,12 @@ def specialization_about(request, specialization_uuid):
                         key=lambda open_rerun: date_from_str(open_rerun['enrollment_start']))[0]
                         if opened else course['course_runs'][0])
         course_rerun['opened'] = opened
-        course_rerun['enrolled'] = CourseEnrollment.is_enrolled(request.user,
-                                                                CourseKey.from_string(course_rerun['key']))
+
+        course_id = CourseKey.from_string(course_rerun['key'])
+        current_course = modulestore().get_course(course_id)
```

Comment:
Lets add a comment why we need to get course from module store and why we cannot use `course_rerun`

Reply:
`get_course_first_chapter_link` requires course object to fetch chapter link thats why we need to get the course from `modulestore`

Reply:
added comment

Reply:
Yes, lets add it as comment, because we have `course_rerun`, which could be used but in actual it is different object with similar name